testnet/stacks-node/src/tests/neon_integrations.rs

```diff
@@ -1310,6 +1310,73 @@ fn bitcoind_forking_test() {
     channel.stop_chains_coordinator();
 }
 
+#[test]
+#[ignore]
+fn should_fix_2771() {
```

Comment:
Do you need to add this test to `Dockerfile.bitcoin-tests`

Reply:
Good catch, probably. Addressed with latest.